scripts/tasks/src/storybook.ts

```diff
@@ -1,9 +1,9 @@
-import { argv } from 'just-scripts';
```

Comment:
seems this was changed by accident, just-scripts are not used in web components and soon enough wont be used in v9 as well. please undo

Reply:
Yeah I wasn't sure how did that file got changed, possibly due to a run of `yarn format`. But I've reverted it. Thanks!